dpgen/data/surf.py

```diff
@@ -23,7 +23,7 @@
 import dpgen.data.tools.fcc as fcc
 import dpgen.data.tools.hcp as hcp
 import dpgen.data.tools.sc as sc
-from dpgen import ROOT_PATH, dlog
```

Comment:
Is removing ROOT_PATH necessary? Is ROOT_PATH used elsewhere?

Reply:
It is not be used